PeapodSwipe/UnitsOfMeasureEnum.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UnitsOfMeasureEnum.swift
+//  PeapodSwipe
+//
+//  Created by Justin Katz on 6/3/18.
+//  Copyright © 2018 Xinjiang Shao. All rights reserved.
+//
+
+import Foundation
+
+enum UOM: String { // enum with type
```

Comment:
Could we try our best to avoid using awkward abbreviations? Using `UnitOfMeasurement` instead of `UOM`, and would be much more readable. Let's just bring less one bad habit from our legacies. :)